packages/sync-actions/test/order-sync.spec.js

```diff
@@ -21,64 +21,321 @@ describe('Actions', () => {
+            ],
+          },
+        ],
+      }
+
+      const now = {
+        returnInfo: [
+          {
+            returnTrackingId: 'touched-item',
+            items: [
+              {
+                id: 'test-1',
+                shipmentState: 'backInStock',
+                paymentState: 'initial',
+              },
+              {
+                id: 'test-2',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+            ],
+          },
+          {
+            returnTrackingId: 'not-touched-item',
+            items: [
+              {
+                id: 'test-3',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+              {
+                id: 'test-4',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+            ],
+          },
+        ],
+      }
+
+      const actual = orderSync.buildActions(now, before)
+      const expected = [
+        {
+          action: 'setReturnShipmentState',
+          returnItemId: now.returnInfo[0].items[0].id,
+          shipmentState: now.returnInfo[0].items[0].shipmentState,
+        },
+      ]
+      expect(actual).toEqual(expected)
+    })
+
+    test('should build `setReturnPaymentState` action', () => {
+      const before = {
+        returnInfo: [
+          {
+            returnTrackingId: 'touched-item',
+            items: [
+              {
+                id: 'test-1',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+              {
+                id: 'test-2',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+            ],
+          },
+          {
+            returnTrackingId: 'not-touched-item',
+            items: [
+              {
+                id: 'test-3',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+              {
+                id: 'test-4',
+                shipmentState: 'returned',
+                paymentState: 'initial',
               },
             ],
           },
         ],
-      },
-    }
-
-    const actual = orderSync.buildActions(now, before)
-    const expected = [
-      {
-        action: 'addDelivery',
-        items: now.shippingInfo.deliveries[0].items,
-        parcels: now.shippingInfo.deliveries[0].parcels,
-      },
-    ]
-    expect(actual).toEqual(expected)
+      }
+
+      const now = {
+        returnInfo: [
+          {
+            returnTrackingId: 'touched-item',
+            items: [
+              {
+                id: 'test-1',
+                shipmentState: 'returned',
+                paymentState: 'initial',
+              },
+              {
+                id: 'test-2',
+                shipmentState: 'returned',
+                paymentState: 'refunded',
```

Comment:
I would add a comment that this is changed, and that this is what you are actually testing... otherwise it is hard to detect/predict what the `actions` should look like